deploy/metrics-server-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metrics-server
+  namespace: kube-system
```

Comment:
This is a replacement for Heapster which runs in `kube-system` and this will run of the the box in every Kubernetes cluster so I think `kube-system` is an appropriate namespace here.